src/compiler/program.ts

```diff
@@ -15,17 +15,23 @@ namespace ts {
     export const version = "1.9.0";
 
     export function findConfigFile(searchPath: string, fileExists: (fileName: string) => boolean): string {
-        let fileName = "tsconfig.json";

```

Comment:
sorry, miss read the code. ignore my comment. `tsc --p jsconfig.json` works as expected.